### ISSUE TITLE
 "Write error" with "Unkown Error" text when database is locked and File | Properties name and descriptions are changed

### ISSUE THREAD

**Issue by igor2x** [pwsafe/pwsafe#1228]
Password Safe v1.19-pre from master build on Ubuntu 23.10

1. Click Lock button to lock database.
2. File  | Properties.
3. Click three dots next to Name or three dots next to Description.
4. Click on Close button. Strange error appears: "Write Error" with "Unknown Error" appears.

It looks like if database is locked then File | Properties can not be changed. Both three buttons should be greyed out if database is Locked.

**Comment by rafaelx** [comment]
In a Wayland (+Gnome) environment, the options dialog can be opened, but this isn't possible in other environments, as the application always has to be unlocked first to regain access to the application and thus to the menu items.

Perhaps the main menu items (except File, due to the unlock menu item) should be disabled to prevent any actions with a locked database.

It's particularly critical that after changing the password for the locked database, no entries are displayed and the database suddenly becomes empty.

**Comment by ronys** [comment]
Good idea, although I'd also leave the Help menu enabled along with File.
